backend/server/src/db/sqlite/mod.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
Empty?

Reply:
yea, I copy and pasted from another PR. I should just delete this for now

Reply:
I removed it for now. sqlite facade/utility can be added later